docs/userGuide/usingPlugins.md

```diff
@@ -1,4 +1,6 @@
+<variable name="title">Using Plugins</variable>
 <frontmatter>

```

Comment:
Put a blank line between `<variable>` and `<frontmatter>`.

Reply:
Fixed!

There is no blank line between `<variable>` and `<frontmatter>` in `tipsAndTricks.md`. Should we fix that in this PR, or make a separate one for that?

Reply:
> Should we fix that in this PR, or make a separate one for that?

I don't mind fixing it in this PR.

Reply:
Then we would need to add a blank line to `tipsAndTricks.md` as well :P